cli/cppcheckexecutor.cpp

```diff
@@ -399,16 +399,18 @@ void CppCheckExecutor::reportOut(const std::string &outmsg, Color c)
         std::cout << c << ansiToOEM(outmsg, true) << Color::Reset << std::endl;
 }
 
+// TODO: remove filename parameter?
 void CppCheckExecutor::reportProgress(const std::string &filename, const char stage[], const std::size_t value)
 {
     (void)filename;
 
     if (!mLatestProgressOutputTime)
         return;
 
-    // Report progress messages every 10 seconds
+    // Report progress messages every x seconds
     const std::time_t currentTime = std::time(nullptr);
-    if (currentTime >= (mLatestProgressOutputTime + 10)) {
+    if (mSettings->reportProgress == 0 || (currentTime >= (mLatestProgressOutputTime + mSettings->reportProgress)))
```

Comment:
This `mSettings->reportProgress == 0 ||` looks redundant.

Reply:
Indeed. That's leftover from when I had some different math for testing. Removed it.